bwakit/test.py

```diff
@@ -8,7 +8,7 @@
 class TestBWAKIT(unittest.TestCase):
 
     def test_docker_call(self):
-        out, err = check_docker_output(tool='quay.io/ucsc_cgl/bwakit:0.7.12')
+        out, err = check_docker_output(tool='quay.io/ucsc_cgl/bwakit')
```

Comment:
the test fails unless I include the full tag (0.7.15--somegithash) which seems like a bad practice.  Without specifying a tag it should grab the latest build, which should be fine as long as "make test" actually does a build